Cocoapods/Networking/Requests/Author/CCPAuthorRequest.m

```diff
@@ -0,0 +1,42 @@
+//
+//  CCPAuthorRequest.m
+//  Cocoapods
+//
+//  Created by Gabriel Massana on 7/2/16.
+//  Copyright © 2016 Gabriel Massana. All rights reserved.
+//
+
+#import "CCPAuthorRequest.h"
+
+@implementation CCPAuthorRequest
+
++ (instancetype)requestToSearchAuthorWithSearchTerm:(NSString *)searchTerm
+{
+    CCPAuthorRequest *request = [self requestForAPI];
+
+    NSMutableString *authorQuery = [[NSMutableString alloc] init];
+    
+    NSArray *terms = [searchTerm componentsSeparatedByString:@" "];
+    
+    for (NSInteger index = 0; index < terms.count; index++)
+    {
+        NSString *term = @"";
+        
+        if (index == 0)
+        {
+            term = [NSString stringWithFormat:@"author:%@", terms[index]];
+        }
+        else
+        {
+            term = [NSString stringWithFormat:@" author:%@", terms[index]];
```

Comment:
It is more about name and username. 
If you search your name in the website, they are producing this URL: https://cocoapods.org/?q=william%20boles

For the API is something similar but with author: before every term:
http://search.cocoapods.org/api/v1/pods.flat.hash.json?query=author:william%20author:boles